src/shared/utils/getNextDebugID.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
We need to add the header document comment here like the other files ([example](https://github.com/facebook/react/blob/master/src/shared/utils/deprecated.js#L1-L13)). 

Without the `@providesModule` comment the haste module system doesn't know where to find this file and tests fail.